frontend/src/libhoclient/host_orchestrator_client.go

```diff
@@ -128,13 +142,19 @@ type OperationsClient interface {
 	WaitForOperation(name string, result any) error
 }
 
+// Pulls service logs via http
+type JournaldClient interface {
```

Comment:
nit: The fact that the ho logs are exposed with journald is an implementation detail. It could change if the ho is running in docker for example. This interface should not include "Journald" in the name.

Also, while not applicable to this PR, the URL where the logs are exposed shouldn't be `.../_journal/entries` but something more general that the implementation (nginx) then redirects or reverse-proxies to journald.

Reply:
Good point.

Rather than committing to a general anstraction to pull service logs, changed the PR to have rather an specific one targeting the [systemd-journal-gatewayd.service](https://www.freedesktop.org/software/systemd/man/latest/systemd-journal-gatewayd.service.html#Supported%20URLs) API, named it `SystemdJournalGatewaydClient` having the `HostOrchestratorClient` interface not including this interface, the implementation does implement the new interface.

Reply:
Moved the implementation out of `HostOrchestratorClientImpl` having a different struct `SystemdJournalGatewaydClientImpl` that implements it.

Reply:
Update PR with offline discussion decision.